### ISSUE TITLE
Offer app to iPad users, but also allow web as well

### ISSUE THREAD

**Issue by jryans** [vector-im/riot-web#12209]
With iPad's recent desktop-class browsing, it's now reasonable to use Riot Web on an iPad directly. However, some users may prefer the native iPad app instead.

We should come up with some not annoying way to offer the iOS app to interested iPad users while still allowing them through to web version that also works well.

**Comment by t3chguy** [comment]
I believe the issue is that the browser claims its a full blown safari and not an IOS variant

**Comment by bwinton** [comment]
> I believe the issue is that the browser claims its a full blown safari and not an IOS variant

Yup. The user-agents for regular and mobile mode are listed in [this comment](https://github.com/vector-im/riot-web/issues/10863#issuecomment-531305722). 🙂
